server/apps/dictionaries/service.py

```diff
@@ -103,6 +103,32 @@ def on_fetched(self, result):
             if 'content' in doc:
                 doc['content'] = decode_dict(doc['content'])
 
+    def get_base_language(self, lang):
+        if lang and lang.find('-') > 0:
+            return lang.split('-')[0]
+
+    def get_dictionaries(self, lang):
+        """
+        Returns all the active dictionaries. If both the language (en-AU)
+        and the base language (en) available, it will return the dict with language
+        :param lang:
+        :return:
+        """
+        languages = [{'language_id': lang}]
+        base_language = self.get_base_language(lang)
+        if base_language:
+            languages.append({'language_id': base_language})
+
+        lookup = {'$and': [{'$or': languages},
+                           {'$or': [{'is_active': {'$exists': 0}}, {'is_active': 'true'}]}]}
+        dicts = list(self.get(req=None, lookup=lookup))
+        langs = [d['language_id'] for d in dicts]
```

Comment:
`langs` could be a `set()` here, but it probably doesn't matter because the expected list length is small.